apps/framework-internal-app/app/flows/UserActivity/ParsedActivity/flow.ts

```diff
@@ -9,7 +9,7 @@ export default function run(event: UserActivity): ParsedActivity {
   // Convert local timestamp to UTC and return new ParsedActivity object.
   return {
     eventId: event.eventId, // Retain original event ID.
-    userId: "puid" + event.userId, // Example: Prefix user ID.
+    userId: "puid2" + event.userId, // Example: Prefix user ID.
```

Comment:
Why puid2? This affects my load test so ideally we don't change this if we don't have to.